services/api-koa/src/resources/account/actions/resend-email.ts

```diff
@@ -43,8 +44,7 @@ async function handler(ctx: AppKoaContext<ValidatedData>) {
   await Promise.all([
     userService.updateOne({ _id: user._id }, () => ({ resetPasswordToken })),
     emailService.sendForgotPassword(user.email, {
-      email: user.email,
-      resetPasswordToken,
+      resetPasswordLink: `${config.webUrl}/reset-password/token=${resetPasswordToken}`
```

Comment:
is it works without ? sign 🤔 
```suggestion
      resetPasswordLink: `${config.webUrl}/reset-password?token=${resetPasswordToken}`
```